tests/test_etcd3.py

```diff
@@ -642,6 +678,35 @@ def test_compact(self, etcd):
         with pytest.raises(grpc.RpcError):
             etcd.compact(1)
 
+    def test_channel_with_no_cert(self):
+        client = etcd3.client(
+            ca_cert=None,
+            cert_key=None,
+            cert_cert=None
+        )
+        assert client.uses_secure_channel is False
+
+    def test_user_pwd_auth(self):
+        auth_mock = mock.MagicMock()
+        auth_resp_mock = mock.MagicMock()
+        auth_resp_mock.token = "the_token"
+        auth_mock.Authenticate = auth_resp_mock
+        etcdrpc.AuthStub = auth_mock
```

Comment:
I don't see this cleaned anywhere. Wouldn't be better to use `with mock.patch …` or something like that?